numpy/core/src/_simd/_simd.c

```diff
@@ -0,0 +1,73 @@
+#include "_simd.h"
+
+PyMODINIT_FUNC PyInit__simd(void)
+{
+    static struct PyModuleDef defs = {
+        .m_base = PyModuleDef_HEAD_INIT,
+        .m_name = "_simd",
```

Comment:
```suggestion
        .m_name = "numpy.core._simd",
```
Assuming that's how this is imported?

Reply:
done, update to match the import path and same goes for the submodules